homeassistant/components/todoist/calendar.py

```diff
@@ -194,14 +180,16 @@ async def handle_new_task(call: ServiceCall) -> None:
         data: dict[str, Any] = {"project_id": project_id}
 
         if task_labels := call.data.get(LABELS):
-            data["label_ids"] = [
-                label_id_lookup[label.lower()] for label in task_labels
-            ]
+            data["labels"] = task_labels
```

Comment:
This key was incorrect per the docs: https://developer.todoist.com/rest/v2/?shell#create-a-new-task  We were also passing the ids of the labels instead of the labels themselves, which the api was expecting.